src/main/java/gregtech/api/items/toolitem/IGTTool.java

```diff
@@ -144,9 +144,39 @@ default ItemStack get(Material material) {
         // Set Material
         toolTag.setString(MATERIAL_KEY, material.toString());
 
+        // Grab the definition here because we cannot use getMaxAoEDefinition as it is not initialized yet
+        AoESymmetrical aoeDefinition = getToolStats().getAoEDefinition(stack);
+
         // Set other tool stats (durability)
         ToolProperty toolProperty = material.getProperty(PropertyKey.TOOL);
-        toolTag.setInteger(MAX_DURABILITY_KEY, toolProperty.getToolDurability());
+
+        // Durability formuala we are working with:
+        // Final Durability = ((material durability * material durability multiplier) + (tool definition durability * definition durability multiplier)) * maximum Area of Effect
+
+        // Calculate the maximum AOE definition of the tool to multiply the final durability by
+        int maxAOEArea = 1;
+        if (aoeDefinition != AoESymmetrical.none()) {
+            maxAOEArea = Math.max(aoeDefinition.column, Math.max(aoeDefinition.row, aoeDefinition.layer));
+
+            // Check for tools with no AOE
+            if (maxAOEArea == 0) {
+                maxAOEArea = 1;
+            }
+        }
```

Comment:
Given that you are doing this check on two places and it is no longer "one liner" it would make sense to create function for it.